src/Microsoft.TestPlatform.Extensions.BlameDataCollector/PublicAPI/PublicAPI.Shipped.txt

```diff
@@ -82,4 +82,4 @@ Microsoft.TestPlatform.Extensions.BlameDataCollector.XmlReaderWriter
 Microsoft.TestPlatform.Extensions.BlameDataCollector.XmlReaderWriter.ReadTestSequence(string filePath) -> System.Collections.Generic.List<Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameTestObject>
 Microsoft.TestPlatform.Extensions.BlameDataCollector.XmlReaderWriter.WriteTestSequence(System.Collections.Generic.List<System.Guid> testSequence, System.Collections.Generic.Dictionary<System.Guid, Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameTestObject> testObjectDictionary, string filePath) -> string
 Microsoft.TestPlatform.Extensions.BlameDataCollector.XmlReaderWriter.XmlReaderWriter(Microsoft.VisualStudio.TestPlatform.Utilities.Helpers.Interfaces.IFileHelper fileHelper) -> void
-override Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameCollector.Initialize(System.Xml.XmlElement configurationElement, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionEvents events, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionSink dataSink, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionLogger logger, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionEnvironmentContext environmentContext) -> void
\ No newline at end of file
+override Microsoft.TestPlatform.Extensions.BlameDataCollector.BlameCollector.Initialize(System.Xml.XmlElement configurationElement, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionEvents events, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionSink dataSink, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionLogger logger, Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection.DataCollectionEnvironmentContext environmentContext) -> void
```

Comment:
Safe to do, I did it few times in the past as well.. This should not even be public api.